benchmarks/zlib_zlib_uncompress_fuzzer/project.yaml

```diff
@@ -0,0 +1,19 @@
+homepage: "http://www.zlib.net/"
```

Comment:
Is this a copy of https://github.com/google/oss-fuzz/blob/master/projects/zlib/project.yaml ?
Why are we checking it into fuzzbench?

Reply:
Yes.
Good question, let's discuss this issue over here: https://github.com/google/fuzzbench/pull/306#discussion_r441220851